datadog-opentelemetry/src/core/configuration/configuration.rs

```diff
@@ -1011,6 +1077,16 @@ impl Config {
             &self.remote_config_enabled,
             &self.remote_config_poll_interval,
             &self.datadog_tags_max_length,
+            &self.otlp_endpoint,
```

Comment:
is metrics_otel_enabled missing?
```suggestion
            &self.metrics_otel_enabled,
            &self.otlp_endpoint,
```